tests/e2e/tailscale/Vagrantfile

```
@@ -0,0 +1,87 @@
+ENV['VAGRANT_NO_PARALLEL'] = 'no'
+NODE_ROLES = (ENV['E2E_NODE_ROLES'] ||
+  ["server-0", "agent-0" ])
+NODE_BOXES = (ENV['E2E_NODE_BOXES'] ||
+  ['generic/ubuntu2004', 'generic/ubuntu2004'])
+GITHUB_BRANCH = (ENV['E2E_GITHUB_BRANCH'] || "master")
+RELEASE_VERSION = (ENV['E2E_RELEASE_VERSION'] || "")
+NODE_CPUS = (ENV['E2E_NODE_CPUS'] || 2).to_i
+NODE_MEMORY = (ENV['E2E_NODE_MEMORY'] || 2048).to_i
+# This key must be created using tailscale web
+TAILSCALE_KEY = (ENV['E2E_TAILSCALE_KEY'] || "")
+NETWORK4_PREFIX = "10.10.10"
+install_type = ""
+
+def provision(node, roles, role_num, node_num)
+  node.vm.box = NODE_BOXES[node_num]
+  node.vm.hostname = "#{roles[0]}-#{role_num}"
+  node_ip4 = "#{NETWORK4_PREFIX}.#{100+node_num}"
+  node.vm.network "private_network", 
+    :ip => node_ip4,
+    :netmask => "255.255.255.0",
+    :libvirt__dhcp_enabled => false,
+    :libvirt__forward_mode => "none"
```

Do we need the special libvirt flags? Its not dualstack
```suggestion
 vm.network "private_network", ip: node_ip, netmask: "255.255.255.0"
```

---

oh, I think we are using those flags in all the networking tests. We should remove it when not needed then. Thanks!